model/team.py

```diff
@@ -49,3 +49,11 @@ def get_members(self):
     def is_member(self, slack_id):
         """Identify if any member has the ID specified in the argument."""
         return slack_id in self.__members
+
+    def __self__(self):
+        """Prints information on the team class."""
+        return "Team, github name: {}, display name: {}. platform: {}" \
```

Comment:
A teensy tiny formatting thing: for this (and for user) I would personally prefer if it printed something like "Team - Github name: {}, display name: {}, ..." (that is, instead of a comma after "Team").

Reply:
Or would something like `return str(self.__dict__)` be better because of the jsonish format? Just a thought.

Reply:
True @rwblickhan would you prefer to use `return str(self.__dict__)` instead?

Reply:
Up to you two - that would definitely look a bit clearer code-wise.

Reply:
I think it would look clearer